### PR TITLE
include signature in id/hash

### PR DESCRIPTION
**Pull request by FabianSimtlix**

### CODE REVIEW

File: src/coin/cgld/types.ts [hunk 1]
```diff
@@ -188,7 +188,7 @@ export class CgldTransactionData implements EthLikeTransactionData {
       gasLimit: new BigNumber(bufferToHex(this.tx.gasLimit), 16).toString(10),
       value: this.tx.value.length === 0 ? '0' : new BigNumber(bufferToHex(this.tx.value), 16).toString(10),
       data: bufferToHex(this.tx.data),
-      id: addHexPrefix(bufferToHex(this.tx.hash())),
+      id: addHexPrefix(bufferToHex(this.tx.hash(true))),
```
Comment (marktoda) [inline src/coin/cgld/types.ts:191]: What if there is no signature yet? Shouldn't it check if there is `v`, `r`, and `s` set and set `useSignature` based on that?

Comment (marktoda) [reply src/coin/cgld/types.ts:191]: @FabianSimtlix ??

Comment (FabianSimtlix) [reply src/coin/cgld/types.ts:191]: I didn't saw this comment.
If the transaction is not signed yet `v` contains the chain id. `r`, `s` has default empty values.

Comment (marktoda) [reply src/coin/cgld/types.ts:191]: Ah so whats even the point of the `includeSignature` flag then

Comment (FabianSimtlix) [reply src/coin/cgld/types.ts:191]: Is for when you verify the signature, there you need a hash that does not contains the signature part

**Pull request merged**